draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -80,15 +80,29 @@
 	<workgroup>Internet Engineering Task Force (IETF)</workgroup>
 	<abstract>
         <t> This document describes how to extend the Internet Key Exchange Protocol
-		Version 2 (IKEv2) to allow multiple key exchanges to take place 
+	Version 2 (IKEv2) to allow multiple key exchanges to take place 
         while computing a shared secret during a Security Association (SA) setup.
-        The primary application of this feature in IKEv2 is the ability to perform one or more 
-        post-quantum key exchanges in conjunction with the classical (Elliptic Curve) Diffie-Hellman key exchange,
+	</t>
+	
+        <t> The primary application of this feature in IKEv2 is the ability to perform one or more 
+        post-quantum key exchanges in conjunction with the classical (Elliptic Curve) Diffie-Hellman (EC)DH key exchange,
         so that the resulting shared key is resistant against quantum computer attacks.
-        Another possible application is the ability to combine several key exchanges 
+	Since there is currently no post-quantum key exchange that is trusted at
+        the level that (EC)DH is trusted for against conventional (non-quantum)
```

Comment:
```suggestion
        the level that (EC)DH is trusted against conventional (non-quantum)
```

Reply:
Actually, this text was suggested by Paul, I copy-pasted it, but the "for" here looked strange and erroneous to me. But I thought that it might be some rare English grammar rule I was not aware of :-)